fe/lib/components/JobCategorySelect/JobCategorySelectInput.tsx

```diff
@@ -97,7 +108,11 @@ const JobCategorySelectInput = ({
     <Stack space="none">
       <Dropdown
         {...restProps}
-        ref={parentRef}
+        ref={(node) => {
```

Comment:
As a side-effect of removing the use-effect the `parentRef` logic no longer works. This was previously reliant on the use-effect causing a re-render.

As that re-render no longer occurs we are intentionally forcing one via setting a state.